app/frontend/src/features/profile/edit/ProfileForm.tsx

```diff
@@ -56,6 +62,12 @@ const useStyles = makeStyles((theme) => ({
     "& > .MuiInputBase-root": {
       width: "100%",
     },
+    "& .MuiOutlinedInput-notchedOutline": {
+      borderColor: theme.palette.grey[500],
+    },
+    "& .MuiOutlinedInput-root:hover .MuiOutlinedInput-notchedOutline": {
+      borderColor: theme.palette.grey[900],
+    },
```

Comment:
Is this to match the hifi more closely? I'm not sure it's 100% necessary, but if we want this, I think it should be in mui theme overrides or in a component - unless these colours shouldn't apply on other pages

Reply:
This is currently in [TextField](https://github.com/Couchers-org/couchers/blob/24e81c4757e6079907c27713a2c78167b667b3fd/app/frontend/src/components/TextField.tsx#L17-L22), but reversed. `TextField` is used a bunch throughout the UI so I didn't want to change it and mess things up

Reply:
I think if this is the correct way, then change them all! :D